src/UserFieldsCreateTemplate.php

```diff
@@ -25,17 +25,14 @@ function addInputItem( $name, $value, $type, $msg ) {
 
 	function execute() {
 		global $sfgTabIndex;
-
-		if ( !$this->haveData( 'header' ) ) {
-			return '';
-		}
+		global $wgOut; /* No other way to access http://www.mediawiki.org/wiki/Manual:$wgOut */
+		$wgOut->addModules( 'ext.semanticforms.main' );
 
```

Comment:
I see what's going on. It's available in MW master but not in MW 1.23 (the one I'm testing). I'd include some conditionals there.